### PR TITLE
[Feature] Tweepy 4.0.0 support

### PR DESCRIPTION
**Pull request by spdconvos**
Change code to be able to use Tweepy>=4.0.0
Any previous version of the bot will not work with Tweepy>=4.0.0 and no Tweepy version <=4.0.0 will work with the bot going forward (run your `pip --upgrades`).

### CODE REVIEW

**Comment by spdconvos** [comment]
This [change](https://github.com/tweepy/tweepy/commit/761cbfe05e866e60edf7c4e58b9ecf356507c6f2) confused me, I think this [commit](https://github.com/spdconvos/encryptedbot_py/commit/ea5178b9c4f5fed846d1e0f019afe3f85c8574cf) addresses the change but I'm not sure??? Please double check.

File: Bot.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
 from datetime import datetime, timedelta
 from typing import List
 import tweepy, json, pytz, socketio
-from tweepy.error import TweepError
+from tweepy.errors import Unauthorized as tweepy_unauthorized
```
Comment (AetherUnbound) [inline Bot.py]: Maybe:
```suggestion
from tweepy.errors import Unauthorized as TweepyUnauthorized
```

File: Bot.py [hunk 2]
```diff
@@ -58,12 +58,9 @@ def __init__(self) -> None:
...
             try:
-                self._api.rate_limit_status()
-            except TweepError as e:
-                if e.api_code == 215:
-                    log.error("No keys or bad keys")
-                else:
-                    log.error("Other API error: {}".format(e))
+                self._api.verify_credentials()
+            except tweepy_unauthorized as e:
+                log.error("No keys or bad keys")
```
Comment (AetherUnbound) [inline Bot.py]: Might be good to log the exception here too:
```suggestion
                log.error(f"No keys or bad keys: {e}")
```

File: Bot.py [hunk 3]
```diff
@@ -58,12 +58,9 @@ def __init__(self) -> None:
...
             # Test the authentication. This will gracefully fail if the keys aren't present.
             try:
-                self._api.rate_limit_status()
-            except TweepError as e:
-                if e.api_code == 215:
-                    log.error("No keys or bad keys")
-                else:
-                    log.error("Other API error: {}".format(e))
+                self._api.verify_credentials()
+            except tweepy_unauthorized as e:
```
Comment (AetherUnbound) [inline Bot.py]: See above:
```suggestion
            except TweepyUnauthorized as e:
```

File: Bot.py [hunk 4]
```diff
@@ -162,10 +159,10 @@ def _postTweet(self, call: dict) -> None:
                         self._cachedTweet = self._api.update_status(msg).id
                     else:
                         self._cachedTweet = self._api.update_status(
-                            msg, self._cachedTweet
+                            msg, in_reply_to_status_id=self._cachedTweet
```
Comment (AetherUnbound) [inline Bot.py:162]: This change looks appropriate with the other changes in the commit you referenced!

**Review by AetherUnbound — COMMENTED** [review]
Couple of minor comments, nice work!

File: Bot.py (hunk 4 above)
Comment (spdconvos) [reply Bot.py:162]: 👍🏻

**Pull request merged**